src/app/server/Mdns.cpp

```diff
@@ -244,25 +232,50 @@ CHIP_ERROR Advertise(bool commissionableNode)
     return mdnsAdvertiser.Advertise(advertiseParameters);
 }
 
+/// Set MDNS commissioner advertisement
+CHIP_ERROR AdvertiseCommisioner()
+{
+    return Advertise(false, CommissioningMode::kDisabled);
```

Comment:
add a comment of  what false is here and true below (like `Advertise(false /* commisionableNode */, ...)`).

Bool arguments are often non-obvious. Code reading I would expect 'Advertise(false)' to mean "do not advertise". Can we fix this readability problem?

Reply:
fixed